tests/contrib/test_lint.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
```

Comment:
Yes, sure.

I'm not sure that tests are run in a file where there is no "test functions".
So I will have to use the first method. What about the `assert True`, do you feel it is okay? (I don't really have better ideas.)